config.json

```diff
@@ -1850,26 +1850,32 @@
   ],
   "key_features": [
     {
+      "icon": "fast",
       "title": "Performance",
       "content": "Rust is blazingly fast and memory-efficient; it has no runtime or garbage collector."
     },
     {
+      "icon": "stable",
```

Comment:
![`stable`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/stable.svg) `stable`

You may prefer e.g.:

![`immutable`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/immutable.svg) `immutable`